src/api/composition-api-dependency-injection.md

```diff
@@ -52,12 +52,16 @@ Injects a value provided by an ancestor component or the application (via `app.p
   function inject<T>(key: InjectionKey<T> | string): T | undefined
 
   // with default value
-  function inject<T>(key: InjectionKey<T> | string, defaultValue: T): T
+  function inject<T>(
```

Comment:
**question**: I am not sure why do we add a third parameter here if we listed the factory below 🤔